hw4task3.py

```diff
@@ -0,0 +1,13 @@
+long_string = "name=Amanda=sssss&age=32&&salary=1500&currency=euro"
+for smtng in long_string:
+    if "=" in long_string:
+        new_string = long_string.replace("=", " ")
+    if "&" in long_string:
+        new_string_2 = new_string.replace("&", " ")
+    if "s" in new_string_2:
+        new_string_3 = new_string_2.replace("s", "", 5)
+    if " " in new_string_3:
+        new_list = new_string_3.split()
```

Comment:
except this loop which is not required at all you can write:

filtered = long_string.replace("=", " ").replace("&", " ").replace("s", "", 5).split()